distributions/distribution-resources/src/main/resources/deb/systemd/openhab2.service

```diff
@@ -1,6 +1,7 @@
 [Unit]
-Description=Starts and stops the openHAB Home Automation Bus
-Documentation=http://www.openhab.org
+Description=openHAB 2 Home Automation Bus Solution
```

Comment:
In general yes, but here I would say it is a fixed term as this is [how it appears in the logo](http://www.openhab.org/assets/images/openhab-logo-top.png).